com.unity.render-pipelines.high-definition/Editor/Material/StackLit/ShaderGraph/StackLitGUI.cs

```diff
@@ -12,15 +12,16 @@ namespace UnityEditor.Rendering.HighDefinition
     class StackLitGUI : HDShaderGUI
     {
         // For surface option shader graph we only want all unlit features but alpha clip and back then front rendering
-        const SurfaceOptionUIBlock.Features   surfaceOptionFeatures = SurfaceOptionUIBlock.Features.Unlit
+        const SurfaceOptionUIBlock.Features   surfaceOptionFeatures = SurfaceOptionUIBlock.Features.Lit
             ^ SurfaceOptionUIBlock.Features.AlphaCutoffThreshold
-            ^ SurfaceOptionUIBlock.Features.BackThenFrontRendering
```

Comment:
we have lost BackThenFrontRendering ?

Reply:
No, I changed `SurfaceOptionUIBlock.Features.UnLit` to `SurfaceOptionUIBlock.Features.Lit` and it includes BackThenFrontRendering